### PR TITLE
refactor(@schematics/angular): changed casings and small reorganizing

### PR DESCRIPTION
**Pull request by fALKENdk**
Updated casings so all categories starts with a capital letter and did some small reorganizations

### CODE REVIEW

File: packages/schematics/angular/workspace/files/__dot__gitignore.template [hunk 1]
```diff
@@ -1,46 +1,45 @@
...
 
-# dependencies
+# Node
 /node_modules
+npm-debug.log
+yarn-error.log
 
-# profiling files
+# Profiling files
 chrome-profiler-events*.json
```
Comment (alan-agius4) [inline packages/schematics/angular/workspace/files/__dot__gitignore.template:15]: This can be removed since we no longer generate this file.

**Review by alan-agius4 — APPROVED** [review]
Thanks, one small NIT

**Comment by fALKENdk** [comment]
Hope I did it correct :innocent:

**Comment by alan-agius4** [comment]
@fALKENdk, you did!

**Pull request merged**